docs/release-notes/Percona-Everest-1.1.0-(2024-08-08).md

```diff
@@ -0,0 +1,127 @@
+# What's new in Percona Everest 1.1.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source cloud native database platform that helps provision and manage databases faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 1.1.0 introduces the following changes:
+
+
+## Release highlights
+
+!!! warning
+    Percona Everest 1.1.0 comes with certain [limitations](Percona-Everest-1.1.0-(2024-08-02).md#known-limitations) that you need to take into account before you update to this version.
+
+### Enhancements in the functionality for PostgreSQL backups and schedules
+
+We've improved the robustness of our backups and restores by enforcing a few limitations in the way we manage backup storage. This proactive approach ensures that we can prevent potential issues from being triggered in edge-case scenarios.
+
+Check out these improvements:
```

Comment:
Should we say that there are the limitations imposed instead of improvements?

Reply:
Instead of 
> Check out these improvements:

say something else regarding limitations. WDYT?

Reply:
We should not put limitations as part of Rel highlights. We should not highlight our limitations. Let's just remove this line altogehter.

Reply:
In that case maybe it doesn't make sense to list the 4 bullets below at all

Reply:
> We've improved the robustness of our backups and restores by enforcing a few limitations in the way we manage backup storage. This proactive approach ensures that we can prevent potential issues from being triggered in edge-case scenarios.

This looks good enough to me